nodejs/aws-serverless/tsconfig.json

```diff
@@ -15,11 +15,9 @@
         "strictNullChecks": true
     },
     "files": [
-        "bucket.ts",
-        "function.ts",
         "index.ts",
-        "topic.ts",
-        "utils.ts",
+        "s3/index.ts",
+        "s3/bucket.ts"
```

Comment:
This looks like it's still missing many files?

Reply:
i'm not sure what our policy is here.  really, we only need to reference index.ts.  TypeScript then figures out the rest.  We only need to specify other .ts files if they were somehow unreachable from the root...